lib/events/auditlog.go

```diff
@@ -348,6 +349,7 @@ func (l *AuditLog) UploadSessionRecording(r SessionRecording) error {
 	}
 	l.log.WithFields(log.Fields{"duration": time.Since(start), "session-id": r.SessionID}).Debugf("Session upload completed.")
 	return l.EmitAuditEventLegacy(SessionUploadE, EventFields{
+		EventID:        uuid.New().String(),
```

Comment:
I'm 95% certain this code is never called and I will be deleting it soon.

Are you sure this was part of the bug or did you just do a global search and see it missing here too?

Reply:
I did check multiple places for `session.start` that I know are used and part of the bug but I'm not sure if/how to trigger the legacy upload if it's used. I saw your comment about `session.upload` missing it and did a global search on that.

Reply:
Ok cool, I have no concerns with making this change here - was just wondering if you actually saw it being called.